src/local/butler/scripts/migration/jobs_keywords.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Build keyword attributes for every job."""
+
+import sys
+
+from datastore import data_types
+from google.cloud import ndb
+
+
+def execute(args):
+  """Build keywords for jobs."""
+  jobs = list(data_types.Job.query())
+  if args.non_dry_run:
+    try:
+      ndb.put_multi(jobs)
```

Comment:
would `ndb_utils.put_multi` be a better choice here? it does batching to avoid the max keys limitation, so you shouldn't need the loop inside the exception handler

Reply:
That was for an older version of ndb which had a bug in the implementation. Should be no longer needed here :) 

That being said.. what is the exception handling for? Did you run into actual issues here?

Reply:
@oliverchang thanks for the clarification. No I didn't run into any actual issues. I add exception handling because it was used in build_attributes.py, which serves a similar purpose. I will remove the for loop and just keep a print message incase something goes wrong.